### PR TITLE
Hover styling based on CodePen in #240

### PR DESCRIPTION
**Pull request by karmatosed**
First pass at a fix for #240 based on the work for by @hugobaeta

### CODE REVIEW

**Comment by karmatosed** [comment]
@melchoyce I would very much like your design review on this as tried to bring in the darker color too.

**Comment by melchoyce** [comment]
Can we inherit the `border-color` from the text `color` using `currentColor`? I know Hugo's mocks used `#000`, but I want to stick to pre-existing colors.

**Comment by melchoyce** [comment]
Looks like no animation on post titles: https://cloudup.com/cYct-ee7DVa

**Comment by melchoyce** [comment]
Seeing it applied on post nav as well:

image

File: style.css [hunk 1]
```diff
@@ -815,8 +813,8 @@ a .nav-title,
 .site-footer .widget-area ul li a,
 .site-footer .widget-area ul li a:visited {
 	border: 0;
-	-webkit-transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
-	transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
+	box-shadow: inset 0 0 0 rgba( 0, 0, 0, 0), 0 0px 0 rgba(0,0,0, 0);
```
Comment (josephfusco) [inline style.css:816]: Remove space after first ( in rgba() and add space after each comma.

It should be

```
box-shadow: inset 0 0 0 rgba(0, 0, 0, 0), 0 0px 0 rgba(0, 0, 0, 0);
```

File: style.css [hunk 2]
```diff
@@ -783,9 +781,9 @@ a:active {
 .site-footer .widget-area a,
 .posts-navigation a,
 .widget_authors a strong {
-	border-bottom: 2px solid #767676;
-	-webkit-transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
-	transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
+	text-decoration: none;
+	box-shadow: inset 0 -1px 0 rgba( 15, 15, 15, 1.0);
```
Comment (josephfusco) [inline style.css:785]: Remove space after opening ( in rgba() and 1.0 can just be 1 here.

Should be 

```
box-shadow: inset 0 -1px 0 rgba(15, 15, 15, 1);
```

File: style.css [hunk 3]
```diff
@@ -841,9 +839,8 @@ a .nav-title,
 .widget_authors a:hover strong,
 .project-terms a:focus,
 .project-terms a:hover {
-	border-bottom-color: transparent;
-	background-color: #767676;
-	color: #fff;
+	color: #000;
+	box-shadow: inset 0 0 0 rgba( 0, 0, 0, 0), 0 3px 0 rgba(0,0,0, 1);
```
Comment (josephfusco) [inline style.css:843]: Same formatting here as well.

``` css
box-shadow: inset 0 0 0 rgba(0, 0, 0, 0), 0 3px 0 rgba(0, 0, 0, 1);
```

File: style.css [hunk 4]
```diff
@@ -876,8 +873,8 @@ a:hover .nav-title,
 .widget .widget-title a:hover,
 .widget ul li a:focus,
 .widget ul li a:hover {
-	background-color: #767676;
-	color: #fff;
+	color: #000;
+	box-shadow: inset 0 0 0 rgba( 0, 0, 0, 0), 0 3px 0 rgba(0,0,0, 1);
```
Comment (josephfusco) [inline style.css:877]: Last one ;)

``` css
box-shadow: inset 0 0 0 rgba(0, 0, 0, 0), 0 3px 0 rgba(0, 0, 0, 1);
```

**Comment by karmatosed** [comment]
My second PR went into new one so closing this in light of: https://github.com/WordPress/twentyseventeen/pull/341

**Pull request closed without merging**